packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,137 @@
+# Nimble Accordion
+
+## Overview
+
+The `nimble-accordion` is a vertical stack of interactive headings. The headings are controls that can be used to reveal or hide content in specific sections. The Nimble accordion is based upon [FAST's accordion component](https://github.com/microsoft/fast/tree/archives/fast-element-1/packages/web-components/fast-foundation/src/accordion).
+
+### Background
+
+[Nimble issue #533: nimble-accordion Component](https://github.com/ni/nimble/issues/533)
+
+[Visual Design](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-85131&mode=design&t=DxDRlqT7MeCPLOxi-0)
+
+---
+
+## Design
+
+Simple accordion group
```

Comment:
nit: some suggestions to improve these examples:
1. I don't think a client would have a use case for an accordion item with no content in the default slot of an item: they'd probably just remove the item in that case. So I'd prefer if the examples didn't emphasize that case so much: right now only 2 of the items have any content in the default slot.
2. When the default slot has content, it's likely to be more complex than just a single direct child. For example it could be a `div` that contains lots of controls, like in a settings pane. 

Could you update the examples to better reflect these use cases?